config-linux.md

```diff
@@ -625,6 +625,22 @@ The following parameters can be specified to set up seccomp:
     * `SECCOMP_FILTER_FLAG_LOG`
     * `SECCOMP_FILTER_FLAG_SPEC_ALLOW`
 
+* **`listenerPath`** *(string, OPTIONAL)* - specifies the path of UNIX domain socket over which the runtime will send the [container process state](#containerprocessstate) data structure when the `SCMP_ACT_NOTIFY` action is used.
+    This socket MUST use `AF_UNIX` domain and `SOCK_STREAM` type.
+    The runtime MUST send exactly one [container process state](#containerprocessstate) per connection.
+    The connection MUST NOT be reused and it MUST be closed after sending a seccomp state.
+    If sending to this socket fails, the runtime MUST [generate an error](runtime.md#errors).
+    If the `SCMP_ACT_NOTIFY` action is not used this value is ignored.
+
+    The runtime sends the following file descriptors using `SCM_RIGHTS` and set their corresponding indexes in the `fdIndexes` map of the [container process state](#containerprocessstate):
+
+    * **`seccompFd`** (int, REQUIRED) is the seccomp file descriptor returned by the seccomp syscall.
+    * **`pidFd`** (int, OPTIONAL) is the process file descriptor (e.g as returned by `pidfd_open(2)` or by `clone(2)` with the `CLONE_PID` flag).
```

Comment:
This is the pidfd of the container's pid 1?

Reply:
Either the container's pid 1, or the pid that entered the container with `exec` (`runc-exec`/`docker-exec`/`kubectl-exec`).

Reply:
Maybe we can add that to the description? Something like: `This refers to the container's pid 1 if the container is being created or to the process that entered the container with exec`.

Would that make it clearer?

Reply:
It seems good to me to add this description.

Reply:
Added it!